### ISSUE TITLE
comments

### ISSUE THREAD

**Issue by anntzer** [tacaswell/CZI_2019-07_mpl#2]
```patch
diff --git i/CZI_mpl.tex w/CZI_mpl.tex
index 716bf39..d8a7830 100644
--- i/CZI_mpl.tex
+++ w/CZI_mpl.tex
@@ -21,9 +21,9 @@ the visualization challenges of the coming decade and reduce backlog
 of currently open Issues and Pull Requests.
 
 By most measures Matplotlib is a wildly successful project; the source
-has been actively developed and maintained by a vibrant primarily
+has been actively developed and maintained by a vibrant, primarily
 volunteer community over the last 16 years and, conservatively, has
-over a million users across the wide spectrum of fields, including
+over a million users [source?] across a wide spectrum of fields, including
 bio-medical imaging, microscopy, and genomics
 \cite{Carpenter2006,Wolf2018,10.7717/peerj.453}
 \cite{Segata2011,10.1371/journal.pgen.1000695,HASHIMSHONY2012666,
@@ -52,11 +52,11 @@ next 16 years:
 % belong here at all.  This is not a goal, but we are in the goals
 % section.  This belongs in the work plan section.
 We propose to fund Thomas Caswell (PI) for 6 months, Hannah Aizenman
-for 12 months, and a yet-unidentified software engineer for 12 months to
+for 12 months, and a yet-unidentified [to be recruited?] software engineer for 12 months to
 work on these tasks.  Caswell is the lead developer of Matplotlib and
 an Associate Computational Scientist at Brookhaven National
 Laboratory.  Aizenman is a core contributor to Matplotlib, who has
-previously worked on datatypes and outreach; she is a PhD student in
+previously worked on datatypes and outreach [datatypes and outreach are dissimilar enough that you may want to split the sentence and use two verbs?]; she is a PhD student in
 computer science studying visualization at The City College of New
 York.
 
@@ -69,9 +69,9 @@ volunteers can review them.
 Over the past few years we have merged PRs and closed
 issues at about about [X per month], but about [Y] new issues and PRs
 are opened monthly; currently we have about 1200 open issues and
-300 open PRs.  Among the latter are good contributions and bug fixes
+300 open PRs.  Among the latter are good [useful?] contributions and bug fixes
 that, possibly with additional attention and polish, could improve
-matplotlib for direct users and downstream packages.  The backlog is
+matplotlib [make sure capitalization is coherent] for direct users and downstream packages.  The backlog is
 discouraging for new or occasional contributors, and distracting for
 core developers.
 
@@ -84,6 +84,10 @@ To maintain matplotlib's health we need to do the following:
   are reviewed in a timely manner.
 \item On-board new contributors to the project team.  This is
   critical to sustaining and diversifying our developer community.
+  [While having more core devs is good, what kind of expansion are you proposing?
+  Something like "for a project of this size, it would be nice to have ~X people
+  with a commit bit, compare with this and that other project", or unbounded
+  (which has its own challenges of coordination -- see last point.)?]
 \item Maintain backward compatibility as much as practical.  If we
   do break API, we must ensure it is intentional and well documented.
 \item Manage group decisions about proposed enhancements, features, and
@@ -125,7 +129,7 @@ plotting tools to be easily built on top of Matplotlib.
 
 ``Structured data'' combines one or more arrays into an object that
 also contains labels, dimensions, and co-ordinates of the data (e.g., a
-map of global surface temperatures containing \texttt{T} ,
+map of global surface temperatures containing \texttt{temperature} ,
 \texttt{latitude} and \texttt{longitude} being packed together in an
 object along with all the relevant metadata).  In addition to the
 data itself, these objects carry information about the
@@ -195,7 +199,7 @@ things like the whole \texttt{Figure}.  In some cases the mapping
 between the user API and the resulting \texttt{Artist}s is one-to-one,
 but in other cases one user call may generate many de-coupled
 \texttt{Artist}s, for example, \texttt{errorbar} creates up to 5
-independent \texttt{Artist}s.  If the user wants to update the plot
+independent \texttt{Artist}s (the line connecting the points, the error bars, the error bar caps, etc.).  If the user wants to update the plot
 they must touch all of the objects.  Instead, we should be providing
 the user with a single object on which they can interact with to elide
 the underlying details of how the \texttt{Artist} is rendered.  This
@@ -287,7 +291,7 @@ Quantitatively evaluating maintenance work can be tricky, some Issues
 or PRs can take minutes to review where as others can take days to
 months of effort, however we believe that there is value at looking at
 the net number of new Issues and Pull requests.  We will reduce this
-number, ideally making it negative.  NumPy has had success in
+number, ideally making it negative [jargony -> "with a goal of closing issues and merging PRs faster than they are opened"].  NumPy has had success in
 reversing the ever increasing trends in the number of Issues / Pull
 Requests via paid
 developers \footnote{https://github.com/seberg/numpy\_talk\_plots/blob/master/plots\_used\_in\_talk/issues\_prs\_delta.pdf}.
@@ -296,7 +300,7 @@ developers \footnote{https://github.com/seberg/numpy\_talk\_plots/blob/master/pl
 We will also evaluate and label every open Issue and Pull Request
 determining: assigning an action, a priority, and an estimated
 difficulty.  Once that is done, we will aim to have all new Issues and
-Pull Requests labeled with in 7 days of being opened.
+Pull Requests labeled within 7 days of being opened.
 
 
 \subsubsection{Architecture and down stream engagement}
```

**Comment by tacaswell** [comment]
Thanks @anntzer! Folded in the comments in f0fab59
